### PR TITLE
fractal: update to 5.

### PR DESCRIPTION
**Pull request by AnInternetTroll**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86-64-glibc)

### CODE REVIEW

File: srcpkgs/fractal/template [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 build_helper="rust"
 hostmakedepends="cargo pkg-config glib-devel gettext"
-makedepends="gtk+3-devel libglib-devel openssl-devel gstreamer1-devel
+makedepends="gtk4-devel libglib-devel openssl-devel gstreamer1-devel
  cairo-devel gettext-devel gst-plugins-base1-devel gst-plugins-bad1-devel
- gst1-editing-services-devel gtksourceview4-devel pango-devel gspell-devel
- gmp-devel libsecret-devel libhandy-devel rust-std"
+ gst1-editing-services-devel gtksourceview5-devel pango-devel gspell-devel
+ gmp-devel libsecret-devel libhandy-devel rust-std libadwaita-devel
+ pipewire-devel libshumate-devel xdg-desktop-portal desktop-file-utils clang"
```
Comment (paper42) [inline srcpkgs/fractal/template:12]: if the last 3 dependencies are required, they should be in hostmakedepends

File: srcpkgs/fractal/template [hunk 2]
```diff
@@ -1,25 +1,29 @@
 # Template file for 'fractal'
 pkgname=fractal
-version=4.4.2
-revision=2
+version=5
+revision=1
+# FIXME: Build fails on non 64-bit archs
+# Try again once this patch makes it into a release and into fractal
+# https://gitlab.freedesktop.org/pipewire/pipewire-rs/-/merge_requests/203
```
Comment (classabbyamp) [inline srcpkgs/fractal/template:7]: try this: https://github.com/void-linux/void-packages/pull/46320/files#diff-a96c43938bfb4a64d9cc96f3b023ec7f3e0338bfba1c589fb04bc4e53ff67df4R19-R22

**Comment by cinerea0** [comment]
Fractal 6 has been released, maybe your build problems have been fixed in that version?

**Comment by chrysos349** [comment]
```
# Template file for 'fractal'
pkgname=fractal
version=6
revision=1
build_style=meson
build_helper=rust
hostmakedepends="cargo clang desktop-file-utils gettext glib-devel
 gtk-update-icon-cache pkg-config"
makedepends="gst-plugins-base1-devel gtksourceview5-devel libadwaita-devel
 libshumate-devel pipewire-devel rust-std"
short_desc="Matrix Messaging Client for GNOME"
maintainer="Cameron Nemo <cam@nohom.org>"
license="GPL-3.0-or-later"
homepage="https://wiki.gnome.org/Apps/Fractal"
distfiles="https://gitlab.gnome.org/World/fractal/-/archive/${version}/fractal-${version}.tar.gz"
checksum=a730dffe5fd4859c16e0d84244a82f6ea843d4e464ee04e08fcb5bea61c248f2

if [ "$XBPS_TARGET_WORDSIZE" = 32 ]; then
	broken="uses i64"
fi

post_patch() {
	vsed -i "/xdg-desktop-portal/d" meson.build
	
	if [ "$CROSS_BUILD" ]; then
		vsed -i src/meson.build \
			-e "s%rust_target /%'${RUST_TARGET}' / rust_target /%"
	fi
}
```

**Comment by AnInternetTroll** [comment]
Hi @chrysos349, I appreciate the suggestion for the fractal 6 template.
Would you mind opening a PR with your change and I'll close mine?

**Comment by chrysos349** [comment]
@AnInternetTroll Done --> #48389

**Pull request closed without merging**